### PR TITLE
Added aggregate-sample function to generate random result from query

### PR DESCRIPTION
**Pull request by adlanarifzr**
The ```sample()``` function is used to generate random result based on the given size. It can be used together with eloquent functions like ```where()`` and such.

### CODE REVIEW

**Comment by jenssegers** [comment]
Is this a new Laravel method?

**Comment by Smolevich** [comment]
Laravel doesn't have this function in query builder

File: src/Jenssegers/Mongodb/Query/Builder.php [hunk 1]
```diff
@@ -209,6 +209,15 @@ public function get($columns = [])
         return $this->getFresh($columns);
     }
 
+    /**
+     * @inheritdoc
```
Comment (jenssegers) [inline src/Jenssegers/Mongodb/Query/Builder.php]: @inheritdoc should be replaced with it’s own docblock since it’s not part of Laravel.

**Comment by usamanoman** [comment]
Very interested in this PR. Let's hope it's available soon.

File: src/Jenssegers/Mongodb/Query/Builder.php (hunk 1 above)
Comment (adlanarifzr) [reply src/Jenssegers/Mongodb/Query/Builder.php]: @jenssegers I updated. Kindly recheck.

**Comment by divine** [comment]
Hello!

Could you please take care for conflicting file or we can take a look?

Thank you for your contribution!

**Comment by divine** [comment]
Closed in favor of https://github.com/jenssegers/laravel-mongodb/pull/1967

**Pull request closed without merging**